tests/fsharpqa/Source/ErrorMessages/UnitGenericAbstractType/E_UnitGenericAbstractType1.fs

```diff
@@ -0,0 +1,9 @@
+// #ErrorMessages #UnitGenericAbstractType 
+//<Expects status="error" span="(7,21)" id="FS0017">The member 'Apply : int -> unit' is specialized with 'unit' but 'unit' can't be used as return type of an abstract method parameterized on return type.\.</Expects>
```

Comment:
@enricosada thanks!

I need more time to dig into TESTGUIDE.md but it seems there is a lot of implicit knowledge about how those tests are defined which is undocumented.

I tried to follow stuff around (the wrong xml stuff was copied from another file) without knowing exactly how that works.